dev-tools/eclipse/dot.classpath.xsl

```diff
@@ -54,7 +55,23 @@
           <classpathentry excluding="src" including="conf/**" kind="src" path="lucene/benchmark"/>
         </xsl:if>
       </xsl:for-each>
-      
+	  
```

Comment:
Could you fix the indent here. In Lucene we only allow whitespace, tabs are forbidden anywhere. This looks like it's mixed here (unfortunately I can't see it here)

Reply:
sure!